cli/src/semgrep/semgrep_main.py

```diff
@@ -441,6 +441,7 @@ def main(
     profiler.save("total_time", rule_start_time)
 
     metrics = get_state().metrics
+    metrics.add_ignores(target_manager.ignore_log.unsupported_lang_paths)
```

Comment:
although the `if metrics.is_enabled:` thing doesn't make too much sense… couldn't we keep this next to all the other metrics.add calls anyway for the sake of consistency?

Reply:
Oh shoot this was for debugging too. Fixed now.